util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+#![cfg(feature = "nightly")]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rlp;
+extern crate rand;
+
+use rlp::RlpStream;
+use rand::Rng;
+
+fuzz_target!(|data: &[u8]| {
+    let mut rls = RlpStream::new();
+
+    // Generate random item count to further fuzz append_raw's parsing
+    let mut rng = rand::thread_rng();
+    let rand_item_count = rng.gen_range::<usize>(1 as usize, 4096 as usize);
+
+    let _ = rls.append_raw(data, rand_item_count);
```

Comment:
trying to get full fuzz-coverage of the classes in the rlp crate, and I hadn't considered where the data may be coming from.

I agree it makes more sense to fully fuzz the UntrustedRlp struct, and am focusing more on that now that the foundational stuff is fixed.

Also still becoming more familiar with the code base, another expectation/goal of writing fuzz tests for all the code.

If for some reason or another, a developer decides to use a normally internal function/struct, it would be beneficial to know if that feature is buggy.